lib/sandbox/pmapi.js

```diff
@@ -253,6 +254,28 @@ function Postman (execution, onRequest, onAssertion, cookieStore, options = {})
         }
     }, options.disabledAPIs);
 
+    _assignDefinedReadonly(this, /** @lends Postman.prototype */ {
+        /**
+         * @interface Execution
```

Comment:
Let's complete the JSDoc with info around what is the purpose of this and what all properties can/should be added here and what should not be added. Will help the next guy.

Reply:
Added